src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/multiplestringliterals/Example2.java

```diff
@@ -8,8 +8,10 @@
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.coding.multiplestringliterals;
+
 // xdoc section -- start
-public class MyClass {
+public class Example2 {
   String a = "StringContents";
```

Comment:
better to have this comment (`// OK, two occurrences are allowed`) at this line instead of the second occurrence at line 19.

since the violation message for Example1 was on the first occurrence not the second. just to keep consistency between examples

Reply:
Yes, please move ok  comment from bottom to first.

Reply:
done